### PR TITLE
vimPlugins.telescope-live-grep-args-nvim: init at 2022-11-07

### PR DESCRIPTION
**Pull request by jocelynthode**
###### Description of changes

telescope-live-grep-args.nvim is a telescope extensions that allows to add grep/rg arguments directly in the search window. It is really useful to search only in a certain filetype for example.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix [hunk 1]
```diff
@@ -172,7 +172,7 @@
       nativeBuildInputs = [ nodejs tree-sitter ];
       postFetch = "pushd $out && tree-sitter generate && popd";
       rev = "c2fbf21bd3aa45495fe13247e040ad5815250032";
-      hash = "sha256-JOCS72Ux1FMzf6kBVO7aqHNwJ0s3xkhzIFFV9GjJVOs=";
+      hash = "sha256-0Nqfg/SiBNj6VrL3npm7LFsujR2dVKg+SNIjDtjToN4=";
```
Comment (figsoda) [inline pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix:175]: looks like there might be a bug in the update script, do you mind reverting the ones that are only hash changes?

Comment (jocelynthode) [reply pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix]: Done :)

**Comment by figsoda** [comment]
It was probably because you were on macos and the `postFetch` messed the hashes up, since I don't think `withPlugins` works on macos atm, I updated the grammars on linux and I will try to fix this issue in a new pr

**Comment by jocelynthode** [comment]
Nope I'm running NixOS unstable.

**Comment by figsoda** [comment]
hmm that is concerning, does `nixpkgs-review rev HEAD` work now?

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 201216` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>169 packages built:</summary>
  <ul>
    <li>vimPlugins.FTerm-nvim</li>
    <li>vimPlugins.SchemaStore-nvim</li>
    <li>vimPlugins.SpaceVim</li>
    <li>vimPlugins.YouCompleteMe</li>
    <li>vimPlugins.aerial-nvim</li>
    <li>vimPlugins.ale</li>
    <li>vimPlugins.alpha-nvim</li>
    <li>vimPlugins.asyncrun-vim</li>
    <li>vimPlugins.asynctasks-vim</li>
    <li>vimPlugins.auto-session</li>
    <li>vimPlugins.bufferline-nvim</li>
    <li>vimPlugins.catppuccin-nvim</li>
    <li>vimPlugins.ccc-nvim</li>
    <li>vimPlugins.chad</li>
    <li>vimPlugins.circles-nvim</li>
    <li>vimPlugins.clangd_extensions-nvim</li>
    <li>vimPlugins.cmp-clippy</li>
    <li>vimPlugins.cmp-cmdline</li>
    <li>vimPlugins.cmp-copilot</li>
    <li>vimPlugins.cmp-dap</li>
    <li>vimPlugins.cmp-dictionary</li>
    <li>vimPlugins.cmp-digraphs</li>
    <li>vimPlugins.cmp-fish</li>
    <li>vimPlugins.cmp-fuzzy-buffer</li>
    <li>vimPlugins.cmp-fuzzy-path</li>
    <li>vimPlugins.cmp-git</li>
    <li>vimPlugins.cmp-greek</li>
    <li>vimPlugins.cmp-look</li>
    <li>vimPlugins.cmp-neosnippet</li>
    <li>vimPlugins.cmp-npm</li>
    <li>vimPlugins.cmp-nvim-lsp-signature-help</li>
    <li>vimPlugins.cmp-nvim-tags</li>
    <li>vimPlugins.cmp-pandoc-nvim</li>
    <li>vimPlugins.cmp-rg</li>
    <li>vimPlugins.cmp-snippy</li>
    <li>vimPlugins.cmp-tabnine</li>
    <li>vimPlugins.cmp-tmux</li>
    <li>vimPlugins.cmp-vim-lsp</li>
    <li>vimPlugins.cmp-vimwiki-tags</li>
    <li>vimPlugins.cmp-zsh</li>
    <li>vimPlugins.coc-fzf</li>
    <li>vimPlugins.coc-lua</li>
    <li>vimPlugins.coc-nvim</li>
    <li>vimPlugins.coc-spell-checker</li>
    <li>vimPlugins.compiler-explorer-nvim</li>
    <li>vimPlugins.completion-treesitter</li>
    <li>vimPlugins.copilot-vim</li>
    <li>vimPlugins.coq-artifacts</li>
    <li>vimPlugins.coq-thirdparty</li>
    <li>vimPlugins.coq_nvim</li>
    <li>vimPlugins.diffview-nvim</li>
    <li>vimPlugins.doki-theme-vim</li>
    <li>vimPlugins.dracula-nvim</li>
    <li>vimPlugins.dracula-vim</li>
    <li>vimPlugins.dressing-nvim</li>
    <li>vimPlugins.edge</li>
    <li>vimPlugins.editorconfig-nvim</li>
    <li>vimPlugins.editorconfig-vim</li>
    <li>vimPlugins.everforest</li>
    <li>vimPlugins.fcitx-vim</li>
    <li>vimPlugins.fern-vim</li>
    <li>vimPlugins.flit-nvim</li>
    <li>vimPlugins.flutter-tools-nvim</li>
    <li>vimPlugins.formatter-nvim</li>
    <li>vimPlugins.fuzzy-nvim</li>
    <li>vimPlugins.fzf-lua</li>
    <li>vimPlugins.git-blame-nvim</li>
    <li>vimPlugins.gitsigns-nvim</li>
    <li>vimPlugins.gruvbox-material</li>
    <li>vimPlugins.gruvbox-nvim</li>
    <li>vimPlugins.haskell-tools-nvim</li>
    <li>vimPlugins.hotpot-nvim</li>
    <li>vimPlugins.iceberg-vim</li>
    <li>vimPlugins.impatient-nvim</li>
    <li>vimPlugins.kanagawa-nvim</li>
    <li>vimPlugins.kommentary</li>
    <li>vimPlugins.lazy-lsp-nvim</li>
    <li>vimPlugins.lazygit-nvim</li>
    <li>vimPlugins.lean-nvim</li>
    <li>vimPlugins.leap-ast-nvim</li>
    <li>vimPlugins.leap-nvim</li>
    <li>vimPlugins.lexima-vim</li>
    <li>vimPlugins.lsp_signature-nvim</li>
    <li>vimPlugins.luasnip</li>
    <li>vimPlugins.lush-nvim</li>
    <li>vimPlugins.material-nvim</li>
    <li>vimPlugins.mini-nvim</li>
    <li>vimPlugins.neo-tree-nvim</li>
    <li>vimPlugins.neodark-vim</li>
    <li>vimPlugins.neodev-nvim</li>
    <li>vimPlugins.neoformat</li>
    <li>vimPlugins.neogit</li>
    <li>vimPlugins.neorg</li>
    <li>vimPlugins.neotest</li>
    <li>vimPlugins.neovim-ayu</li>
    <li>vimPlugins.nerdcommenter</li>
    <li>vimPlugins.nightfox-nvim</li>
    <li>vimPlugins.noice-nvim</li>
    <li>vimPlugins.nordic-nvim</li>
    <li>vimPlugins.null-ls-nvim</li>
    <li>vimPlugins.nvim-biscuits</li>
    <li>vimPlugins.nvim-bqf</li>
    <li>vimPlugins.nvim-cmp</li>
    <li>vimPlugins.nvim-dap</li>
    <li>vimPlugins.nvim-dap-python</li>
    <li>vimPlugins.nvim-dap-ui</li>
    <li>vimPlugins.nvim-hlslens</li>
    <li>vimPlugins.nvim-lint</li>
    <li>vimPlugins.nvim-lspconfig</li>
    <li>vimPlugins.nvim-metals</li>
    <li>vimPlugins.nvim-notify</li>
    <li>vimPlugins.nvim-scrollbar</li>
    <li>vimPlugins.nvim-scrollview</li>
    <li>vimPlugins.nvim-surround</li>
    <li>vimPlugins.nvim-tree-lua</li>
    <li>vimPlugins.nvim-treesitter</li>
    <li>vimPlugins.nvim-treesitter-textobjects</li>
    <li>vimPlugins.nvim-ts-rainbow</li>
    <li>vimPlugins.nvim-web-devicons</li>
    <li>vimPlugins.nvimdev-nvim</li>
    <li>vimPlugins.octo-nvim</li>
    <li>vimPlugins.onedarkpro-nvim</li>
    <li>vimPlugins.onenord-nvim</li>
    <li>vimPlugins.orgmode</li>
    <li>vimPlugins.refactoring-nvim</li>
    <li>vimPlugins.registers-nvim</li>
    <li>vimPlugins.rnvimr</li>
    <li>vimPlugins.rose-pine</li>
    <li>vimPlugins.rust-tools-nvim</li>
    <li>vimPlugins.satellite-nvim</li>
    <li>vimPlugins.sonokai</li>
    <li>vimPlugins.swift-vim</li>
    <li>vimPlugins.tagbar</li>
    <li>vimPlugins.telescope-cheat-nvim</li>
    <li>vimPlugins.telescope-frecency-nvim</li>
    <li>vimPlugins.telescope-fzf-native-nvim</li>
    <li>vimPlugins.telescope-fzf-writer-nvim</li>
    <li>vimPlugins.telescope-fzy-native-nvim</li>
    <li>vimPlugins.telescope-live-grep-args-nvim</li>
    <li>vimPlugins.telescope-manix</li>
    <li>vimPlugins.telescope-media-files-nvim</li>
    <li>vimPlugins.telescope-nvim</li>
    <li>vimPlugins.telescope-symbols-nvim</li>
    <li>vimPlugins.telescope-z-nvim</li>
    <li>vimPlugins.telescope-zoxide</li>
    <li>vimPlugins.template-string-nvim</li>
    <li>vimPlugins.todo-comments-nvim</li>
    <li>vimPlugins.tokyonight-nvim</li>
    <li>vimPlugins.vim-airline</li>
    <li>vimPlugins.vim-airline-themes</li>
    <li>vimPlugins.vim-anzu</li>
    <li>vimPlugins.vim-clap</li>
    <li>vimPlugins.vim-codefmt</li>
    <li>vimPlugins.vim-dadbod-completion</li>
    <li>vimPlugins.vim-fubitive</li>
    <li>vimPlugins.vim-illuminate</li>
    <li>vimPlugins.vim-lsp</li>
    <li>vimPlugins.vim-markdown</li>
    <li>vimPlugins.vim-matchup</li>
    <li>vimPlugins.vim-mundo</li>
    <li>vimPlugins.vim-ocaml</li>
    <li>vimPlugins.vim-test</li>
    <li>vimPlugins.vim-tmux-navigator</li>
    <li>vimPlugins.vim-vsnip</li>
    <li>vimPlugins.vim-vsnip-integ</li>
    <li>vimPlugins.vim-xkbswitch</li>
    <li>vimPlugins.vimspector</li>
    <li>vimPlugins.vimtex</li>
    <li>vimPlugins.zk-nvim</li>
  </ul>
</details>

**Review by figsoda — APPROVED** [review]
changes lgtm, I can merge this if it works for you

**Comment by jocelynthode** [comment]
It was workng before your commits and it's still working now :)

**Pull request merged**